### ISSUE TITLE
Allow to select the look and feel

### ISSUE THREAD

**Issue by vdun** [zaproxy/zaproxy#2964]
I'm trying to use the GTK look and feel :
```
# export _JAVA_OPTIONS='-Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel'
# zaproxy
Found Java version 1.8.0_112
Available memory: 7970 MB
Setting jvm heap size: -Xmx512m
Picked up _JAVA_OPTIONS: -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel
0 [main] INFO org.zaproxy.zap.GuiBootstrap  - OWASP ZAP 2.5.0 started.
...
```
but still getting the old look and feel!
The trick works on other Jar based applications except ZAP.

**Comment by thc202** [comment]
The L&F used by ZAP is currently "hardcoded".

**Comment by psiinon** [comment]
The relevant code is here: https://github.com/zaproxy/zaproxy/blob/develop/src/org/zaproxy/zap/GuiBootstrap.java#L325,L352
I'd have no problem for this to be changeable via a config option, as long as the default stays the same.

**Comment by nhegde610** [comment]
Hi, 
I want to take up this issue. I have looked through the code and pretty much understood how to fix this issue. However, I don't know which files should I edit to create a config option in the options window. If possible, I would like to get some answers regarding this.

**Comment by thc202** [comment]
I guess this would go under Options > Display, if so, the classes that need to be changed are [`org.parosproxy.paros.extension.option.OptionsViewPanel`](https://github.com/zaproxy/zaproxy/blob/ca3ff4cfb7d34e91ab31ccccfa7ffb03883218df/src/org/parosproxy/paros/extension/option/OptionsViewPanel.java) (this one creates the Display panel) and  `OptionsParamView` (reads and persists the corresponding options).

**Comment by nhegde610** [comment]
Thank you @thc202. I have managed to create a config options in Display Options panel. While I was testing my solution, I noticed a UI bug. When I switch the look to native system look, the option for changing look  disappears. I have to resize the options window to be able to view it. This issue also arises if the user changes the font size and font name. Here are pics for better understanding.
Before resizing :
![before_resizing](https://user-images.githubusercontent.com/18752973/36634438-a8fca258-19ca-11e8-96e8-bc3b8f7f2e73.png)

After resizing: 
![after_resizing](https://user-images.githubusercontent.com/18752973/36634444-c5b6aa6a-19ca-11e8-97e7-065c9e548efa.png)


I am thinking of fixing this issue by adding a scrollbar. Should I add a scrollbar code in the org.parasproxy.paros.extension.option.OptionsViewPanel ? 

Regarding the selection of look and feel, I have added 3 looks : default look, native system look and cross platform look. Shall add I more?

**Comment by thc202** [comment]
Yes, the panel needs a scroll pane :) That can be fixed in the same pull request if you want.

IMO it should allow to select all available, it's not necessary to restrict to just some.

**Comment by nhegde610** [comment]
@thc202 I would like to submit a new pull request with added scroll pane code.

**Comment by kingthorin** [comment]
You can amend the existing one, just push to the same origin branch

**Comment by nhegde610** [comment]
I have amended the existing PR. The changes includes scrollbar too. Please review the code and let me know if I have to make any changes.

**Comment by thc202** [comment]
Fixed in #4438.
